app/pods/publications/publication/template.hbs

```diff
@@ -95,7 +95,13 @@
                   {{/if}}
                 </div>
                 <div class="auk-form-group">
-                  <WebComponents::AuLabel>{{t 'extreme-translation-date'}}</WebComponents::AuLabel>
+                  {{#if this.publicationNotAfterTranslation}}
+                    <div class="auk-form-help-text auk-form-help-text--warning">
+                      <WebComponents::AuIcon @name="alert-triangle"/>
```

Comment:
Dit geeft een warning, een van de design review opmerkingen is dat het een rode driehoek en tekst moet weergeven omdat het een error moet zijn en geen waarschuwing.
De tekst mag ook net onder het invoervak staan en wat marge onderaan hebben (nu is het omgekeerd)
![image](https://user-images.githubusercontent.com/11557630/101799165-1ef5be80-3b0c-11eb-8a26-f46181818e79.png)